ledger/participant-integration-api/src/test/suite/scala/platform/indexer/parallel/BatchingParallelIngestionPipeSpec.scala

```diff
@@ -109,7 +109,7 @@ class BatchingParallelIngestionPipeSpec
     ).map { case (_, _, err) =>
       // The first and last batches can be smaller than `MaxBatchSize`
       // so we assert the average batch size instead of the sizes of individual batches
-      batchSizes.sum.toDouble / batchSizes.size should be > (MaxBatchSize.toDouble - MaxBatchSize / 10.0)
+      batchSizes.sum.toDouble / batchSizes.size should be > (MaxBatchSize.toDouble - MaxBatchSize / 5.0)
```

Comment:
Took me a while until I understood what value we have accepted before and what do we accept now with this change :) 

Maybe this would help readability:
```batchSizes.sum.toDouble / batchSizes.size should be(MaxBatchSize.toDouble +- 1)```

Reply:
It is not the same thing. This test does not always forms max sized batches so we need to allow some room for variation. Here, I've increased that room for variation since it was sometimes flaky